### PR TITLE
add UCSC Xena tool

### PR DESCRIPTION
**Pull request by inodb**
Fix https://github.com/ncihtan/htan-portal/issues/592

image

TODO:
- [x] Currently on Files tab it replaces the imaging thumbnail (want to keep that and put Xena Viewer URL in tooltip instead)
- [x] add Xena Icon on both Atlas & Files Tab
- [ ] It could be better if viewers were just another file attribute rather than mapping them in later, code getting kinda messy (also would be nice to have a "Viewer" filter)

### CODE REVIEW

File: components/tool/toolDetails.tsx [hunk 1]
```diff
@@ -4,12 +4,14 @@ import Mcmicro from './Mcmicro';
 import GoogleBigQuery from './GoogleBigQuery';
 import CellxGene from './CellxGene';
 import CytoCommunity from './CytoCommunity';
+import UCSCXena from './CytoCommunity';
```
Comment (onursumer) [inline components/tool/toolDetails.tsx:7]:
```suggestion
import UCSCXena from './UCSCXena';
```

File: components/tool/UCSCXena.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+                UCSC Xena is an online exploration tool for multiplexed imaging and multi-omic single-cell data. 
+                <a href="https://xena.ucsc.edu/">
+                    <img src="https://github.com/cBioPortal/cbioportal/assets/1334004/0f58a05e-5c4d-42a2-8d0d-61b630dc2e25"
+                    />
+                </a>
+                HTAN melanoma sample t-CyCIF image showing three antibody signals. Genomic data is overlaid as circles, coloring cells annotated as keratinocytes in purple. Users can also view an interactive 3D UMAP/tSNE of the same data. To navigate the tool, select spatial or embedding layouts, manipulate image layers, and choose the genomics data to color the cells.
+            </p>
+        </div>
+    );
+}
```
Comment (onursumer) [inline components/tool/UCSCXena.tsx:21]:
```suggestion
}

export default UCSCXena;
```

**Review by onursumer — CHANGES_REQUESTED** [review]
Looks like we are importing the wrong component for tool details

**Comment by jingchunzhu** [comment]
Using the testing site, the Xena team notice these issues below

1. missing Xena Icon on "Atlas" tab
2. on "Files" tab, when there are linkouts to both "UCSC Xena" and "Minerva",  Minerva imaging thumbnail/linkout disappears
3. missing "UCSC Xena" under "ANALYSIS TOOLS" 

We identified three more HTAN files that can be linked out to Xena. If it is not too much trouble, can those these be added?

HTA7_926_1015  https://beta.xenabrowser.net/singlecell/?hub=https://previewsinglecell.xenahubs.net:443&defaultTable=htan&study=HTA7_926_8

HTA7_934_1016  https://beta.xenabrowser.net/singlecell/?hub=https://previewsinglecell.xenahubs.net:443&defaultTable=htan&study=HTA7_934_9

HTA7_16_1002     https://beta.xenabrowser.net/singlecell/?hub=https://previewsinglecell.xenahubs.net:443&defaultTable=htan&study=HTA7_16_2

**Comment by onursumer** [comment]
@jingchunzhu Thanks for the feedback! We were able to fix the issues you pointed out. Can you do a final check? If everything looks good, we would like to make this available on the data portal.

**Comment by jingchunzhu** [comment]
Looks great on the preview site. Thank you everyone.

**Review by onursumer — APPROVED** [review]

**Pull request merged**